.babelrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This actually causes problems up the build chain if you're using an older version of babel (5). I realize babel 5 is deprecated, but there are those still using it (like my organization). When I tried to babelify this module, it complained about the babel config in the package.json. If you put the config in `.babelrc` then don't include that in the npm package (via `files`), it plays nicely with consumers.